More2Life/DetailViewController.swift

```diff
@@ -53,11 +53,7 @@ class DetailViewController: UIViewController, FeedDetailing {
         switch feedItem {
         case _ as EventFeedItem:
             break
-        case let feedItem as ListingFeedItem:
```

Comment:
Where are we showing the price on this screen if we don't do this?

Reply:
Just showing "Buy"?

Reply:
Good point. I removed the price because i think it's a weird experience to hit the same button twice. A button with "$25.00" that then opens a modal with the variant selection and two more Buy buttons is an awkward flow IMO. But the product detail does need the price somewhere. 

My first inclination is to put the price button over the picture like in the feed but let me think about this one.